api/formatters/geojson.py

```diff
@@ -0,0 +1,63 @@
+from fastapi import HTTPException
+from geojson_pydantic import Feature
+from geojson_pydantic import FeatureCollection
+from geojson_pydantic import Point
+
+
+def _make_properties(ts):
```

Comment:
Do we want to use reflection to avoid having to change this every time the fields change? Or leave it for now in the interest of progress?

Reply:
I have rewritten it.